### PR TITLE
Update Bootstrap.rst

### PR DESCRIPTION
**Pull request by griebie**
I had difficulty understanding exactly what was meant here and hope this is still transmitting the correct content :)

### CODE REVIEW

**Review by linawolf — APPROVED** [review]
Like that it is much clearer, thank you!

File: Documentation/Extbase/Step3Documentation/Bootstrap.rst [hunk 1]
```diff
@@ -32,8 +32,8 @@ global TypoScript template::
    pluginName = Products
  }
 
-As soon as you add this Extbase plugin to a page, the method *run* in the class *tx_extbase_core_bootstrap* is 
-exectued. So that it knows wich extension and plugin we're dealing with, information about the *extensionName* and 
+As soon as you add this Extbase plugin to a page, the method *run* of the class *tx_extbase_core_bootstrap* is being
```
Comment (DavidBruchmann) [inline Documentation/Extbase/Step3Documentation/Bootstrap.rst:35]: I'd say this:
... *run* by the class *tx_extbase_core_bootstrap* ...

Comment (linawolf) [reply Documentation/Extbase/Step3Documentation/Bootstrap.rst:35]: this is merged would you make a new PR about that @DavidBruchmann?

**Comment by DavidBruchmann** [comment]
if it's no big burden for you I'd appreciate if you do it, but else let me know, then I'll do it

**Pull request merged**